cfgmgr/teammgr.cpp

```diff
@@ -520,11 +581,17 @@ task_process_status TeamMgr::addLagMember(const string &lag, const string &membe
 
     stringstream cmd;
     string res;
+    uint16_t keyId = generateLacpKey(lag);
 
     // Set admin down LAG member (required by teamd) and enslave it
     // ip link set dev <member> down;
+    // teamdctl <port_channel_name> port config update <member> { lacp_key: <lacp_key>, link_watch: { name: ethtool } };
     // teamdctl <port_channel_name> port add <member>;
     cmd << IP_CMD << " link set dev " << shellquote(member) << " down; ";
+    cmd << TEAMDCTL_CMD << " " << shellquote(lag) << " port config update " << shellquote(member)
```

Comment:
Is teamd ok ( no errors seen here ) as we are doing a "port config update" before doing actual member add below using "port add". Should we do this call after "port add" ?

Reply:
No, there are no errors printed from teamd when we are running the commands in this error.

Trying to run the port add before the config does results in errors:
> root@arc-switch1038:/home/admin# ip link set dev Ethernet64 down
> root@arc-switch1038:/home/admin# teamdctl PortChannel0005 port add Ethernet64
> root@arc-switch1038:/home/admin# teamdctl PortChannel0005 port config update Ethernet64 '{ "lacp_key": 10005, "link_watch": { "name": "ethtool" } }'
> libteamdctl: cli_usock_process_msg: usock: Error message received: "ConfigUpdateFail"
> libteamdctl: cli_usock_process_msg: usock: Error message content: "Failed to update config."
> command call failed (Invalid argument)